features/support/helpers.rb

```diff
@@ -11,12 +11,38 @@
   PaaS: 'Platform as a Service',
   IaaS: 'Infrastructure as a Service',
   SCS: 'Specialist Cloud Services'
-}
+}.freeze
+
+META_DATA_XPATHS = {
+  location: '//*[@class="search-result"]//*[@class="search-result-important-metadata"][1]//*[@class="search-result-metadata-item"][2]',
+  lot: '//*[@class="search-result"]//*[@class="search-result-metadata"][1]//*[@class="search-result-metadata-item"][1]',
+  role: '//*[@class="search-result"]//*[@class="search-result-metadata"][1]//*[@class="search-result-metadata-item"][2]',
+  status: '//*[@class="search-result"]//*[@class="search-result-metadata"][2]//*[@class="search-result-metadata-item"][1]'
+}.freeze
+
+CLOSED_OUTCOMES = {
+  Awarded: 'Closed: awarded',
+  Cancelled: 'Closed: cancelled',
+  Closed: 'Closed: awaiting outcome',
+  Unsuccessful: 'Closed: no suitable suppliers'
+}.freeze
```

Comment:
Guessing this makes `CLOSED_OUTCOMES` unmutable?

Reply:
Yes indeed :)